src/aics-image-viewer/shared/enums.ts

```diff
@@ -1,17 +1,17 @@
-export const enum ViewMode {
+export enum ViewMode {
```

Comment:
Allows the `parseStringEnum` function to work. `const` enums are removed at runtime, while non-`const` enums persist as objects.